CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - [Track requests and dependencies from ServiceBus .NET Client 2.1.0](https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/740)
 - [Fix for REST API Request filter bug](https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/175)
 - [Fix for SyntheticUserAgentTelemetryInitializer null check](https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/750)
+- [Track dependencies from EventHubs .NET Client 2.0.0](https://github.com/Azure/azure-event-hubs-dotnet/pull/203)
```

Comment:
make it explicit that 2.0.0 version of EventHub/Sb client to avoid potential confusion with .NET itself

Reply:
Done